lib/msf/core/handler/reverse.rb

```diff
@@ -82,12 +82,15 @@ def setup_handler
                 'MsfPayload' => self,
                 'MsfExploit' => assoc_exploit
               })
+	
+	  if (ip=='127.0.0.1' || ip=='127.0.0.2' || ip=='127.0.0.3' || ip=='127.0.0.4'|| ip=='127.0.0.5'||ip=='127.0.0.6' || ip=='127.0.0.7' || 		ip=='127.0.0.8')
```

Comment:
My original suggestion from the issue was to check if this is in the loopback subnet 127.0.0.0/8, not the first 8 addresses. The proper check function would look something like this (I'm sure the rescue could be tightened up, but I prefer test functions to really only return booleans where possible):

```
require 'ipaddr'

def is_loopback_addr(addr)
  begin
    a = IPAddr.new(addr)
    return true if IPAddr.new('127.0.0.1/8') === a
    return true if IPAddr.new('::1') === a
  rescue
  end
  false
end

[IPAddr.new('127.0.0.1'), '127.0.0.1', '127.255.255.255', '::1', 'dog'].each do |addr|
  puts "#{addr} is loopback: #{is_loopback_addr(addr)}"
end
```